.travis.yml

```diff
@@ -38,7 +38,9 @@ jobs:
         cross-env NODE_ENV=production yarn run build
         yarn run typedoc
         cross-env NODE_ENV=production yarn run build-www
-      before_deploy: yarn prepare-doc-deploy
+        git status # just in build leaves some garbage
+        yarn prepare-doc-deploy
+        find dist/
```

Comment:
That is not a good test, as it can still fail even if dist/ is there. I would not add it.

Reply:
Yes, i understand this is only `set -x`-like solution, so we see in logs what was actually produced.